PluginBuilder/Views/Plugin/Settings.cshtml

```diff
@@ -34,6 +90,17 @@
                 <input asp-for="BuildConfig" class="form-control" placeholder="Release" />
                 <span asp-validation-for="BuildConfig" class="text-danger"></span>
             </div>
+
+            <div class="form-group">
```

Comment:
We should add a Remove button too, like we have on the server.

Reply:
Once an image is uploaded there is an 'X' by the side that you can use to remove the uploaded image and then you can reuploaded